### PR TITLE
Add actions api revision migration guide

### PR DESCRIPTION
**Pull request by gspencergoog**
cc @Hixie for review.

### CODE REVIEW

File: src/docs/release/breaking-changes/actions-api-revision.md [hunk 1]
```diff
@@ -0,0 +1,311 @@
...
+          Activate.key: () => ActivateAction(),
+        },
+        child: Container(),
+      )
+    );
+  }
+}
+```
+
+Code after migration:
```
Comment (sfshaza2) [inline src/docs/release/breaking-changes/actions-api-revision.md:134]: Add a  tag before this code. It is currently breaking the build, because our build tests against the stable release (1.12 at the moment).

File: src/docs/release/breaking-changes/actions-api-revision.md [hunk 2]
```diff
@@ -0,0 +1,311 @@
...
+### Actions mapping for pre-defined actions
+
+To update the action maps in the `Actions` widget for predefined actions in
+Flutter, like `ActivateAction` and `SelectAction`, etc., you'll need to:
+
+ - Update the argument type of the actions argument, and
+ - Use an instance of a specific Intent class in the Shortcuts mapping, rather than an Intent(TheAction.key) instance.
+
+Code before migration:
+```dart
```
Comment (sfshaza2) [inline src/docs/release/breaking-changes/actions-api-revision.md:114]: Add a skip tag here because once we upgrade the stable release, the old code will fail (and break the build), yes?

File: src/docs/release/breaking-changes/actions-api-revision.md [hunk 3]
```diff
@@ -0,0 +1,311 @@
...
+        },
+        child: Container(),
+      )
+    );
+  }
+}
+```
+
+### Custom actions
+To migrate your custom actions, you'll need to eliminate the `LocalKeys` you've
```
Comment (sfshaza2) [inline src/docs/release/breaking-changes/actions-api-revision.md:156]: You are using "will" and "you'll" quite a bit. Can you rephrase to make it present tense?

File: src/docs/release/breaking-changes/actions-api-revision.md (hunk 1 above)
Comment (gspencergoog) [reply src/docs/release/breaking-changes/actions-api-revision.md:138]: Done.  I added one before the block of analyzer warnings too, in case it's not looking at the "dart" part of the triple quotes.

File: src/docs/release/breaking-changes/actions-api-revision.md (hunk 3 above)
Comment (gspencergoog) [reply src/docs/release/breaking-changes/actions-api-revision.md]: Sure, I removed all of the instances of both.

File: src/docs/release/breaking-changes/actions-api-revision.md (hunk 2 above)
Comment (gspencergoog) [reply src/docs/release/breaking-changes/actions-api-revision.md:118]: Yes. Done.

**Comment by sfshaza2** [comment]
@gspencergoog, there is now a conflict with the breaking change index file. (Lots of folks are writing them these days.) Can you merge? Is this otherwise good to go? (It seems to be.)

**Comment by Hixie** [comment]
This looks great.
